src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+const ACTIVE_STYLE = 'is-active';
+
+export const Tabs = ({
+  tabs,
+  selectedId,
+  onTabSelected,
+}) => {
+  const className = tabId => ((tabId === selectedId
+    || (!selectedId && tabId === tabs[0].id))
+    ? ACTIVE_STYLE : null);
+
+  return (
+    tabs.map((tab) => {
+      const { id, title } = tab;
+
+      const linkEventHandler = (event) => {
+        event.preventDefault();
+        if (selectedId !== id) {
+          onTabSelected(tab);
+        }
+      };
```

Comment:
It will be better to carry out this logic before the return, and then use this function